### ISSUE TITLE
ValueError: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by gkrisp98** [ly19965/MogFace#9]
I am trying to train the mogface with configurations from MogFace.yml to reproduce the best sota results on wider face but I am getting this error:
MogFace-master/train.py", line 138, in <module>
    images, targets = next(batch_iterator)
ValueError: too many values to unpack (expected 2)

Does anybody know how to overcome this error ?

**Comment by gdiasbruno** [comment]
What command did you execute? Do you pass any custom number for batch size?

**Comment by gkrisp98** [comment]
I execute the following command:
`python train.py -c configs/mogface/MogFace.yml
`
I have tried different numbers fot batch_size but all of them produce the same error

**Comment by gdiasbruno** [comment]
You are running it using which hardware setup? I

 First tried to train using a CPU and I faced issues. But running in a GPU solved them all.

**Comment by gkrisp98** [comment]
I am running it in GPU...
When I run this command:
`train.py -c configs/mogface/MogFace_Ali-AMS.yml` , the model is trained normally and I obtain the appropiate results. I don't know why I am getting the error on the MogFace.yml and MogFace_E.yml

**Comment by gkrisp98** [comment]
Did you make any change on the config files ?

**Comment by gdiasbruno** [comment]
I did, but related to the use case I'm experimenting with. I did start from `MogFace_Ali-AMS.yml`.

Maybe it is a good idea to compare both config files `MogFace_Ali-AMS.yml` and `MogFace.yml` and check what can potentially be the source of the issue.

I do see that in the README they do not have these configs in the example, so perhaps the `train.py` script they provided is not suited to these configs.

**Comment by gkrisp98** [comment]
That's correct but I wanted to get the best results on Widerface (mAP 93%) and MogFace_Ali-AMS is not that close..

**Comment by gdiasbruno** [comment]
Did you find any solution to it?

**Comment by gkrisp98** [comment]
Unfortunately no..
